plugins/outputs/execd/sample.conf

```
@@ -13,6 +13,11 @@
   ## Delay before the process is restarted after an unexpected termination
   restart_delay = "10s"
 
+  ## Flag to determine whether execd should throw error when part of metrics is unserializable
+  ## Setting this to true will skip the unserializable metrics and process the rest of metrics
+  ## Setting this to false will throw error when encountering unserializable metrics and none will be processed
+  ignore_serialization_error = false
```

Defaults can be commented:
```suggestion
  # ignore_serialization_error = false
```

---

I believe I fixed it!